Author: Michael Vogt <michael.vogt@ubuntu.com>
Description: Various default settings
 Changes in various defaults.  From what I can tell (these were originally
 from ubuntu, but I (the debian maintainer) will do my best to summarize
 them below).
 .
 * Desktop Horizontal virtual size of 2 instead of 4.
 * Sync to vblank: false by default.
 * Make an exception to focus prevention matching for PolicyKit windows.
 * Remove top-right hotcorner from scale plugin by default.
 * Increase drop-shadow radius from 8 to 9.
 * Set default decorator to /usr/bin/compiz-decorator.
 * Change default colors for faces in cube plugin.
 * Use "Smart" placement mode (2) by default for placement plugin.
 * Use "Rectangle" resize mode (2) by default for resize plugin.
 * Add exception to not fade notify-osd windows in fade plugin.
 .
 The following changes from the original Ubuntu patch were modified:
 * Use slightly more Debianish/Gnomeish colors for the cube face top.
 The following changes from the original Ubuntu patch were not included:
 * Add Ubuntu logo for top face of cube.
 The following changes were introduced to the Debian version of this patch:
 * Add Debian logo for top face of cube.
Origin: vendor, ubuntu (1:0.6.0+git20071006-0ubuntu1) or perhaps earlier.

Author: Jasmine Hassan
Description: Updated patch for Debian/Mate
 1. use new /usr/bin/compiz-mate-gtk instead of /usr/bin/compiz-decorator
 2. use Polkit-mate-authentication-agent-1 instead of gnome's
 3. revert to debian-blueish-wallpaper-640x480.png instead of ubuntu's to
    adhere with original debian packaging. In LMDE package, this will be
    again replaced with mint-logo.svg in 200-linuxmint-logo.patch.
Last-Updated: 2012-10-09

--- a/metadata/core.xml.in.in
+++ b/metadata/core.xml.in.in
@@ -284,7 +284,7 @@
 		<option name="sync_to_vblank" type="bool">
 		    <_short>Sync To VBlank</_short>
 		    <_long>Only perform screen updates during vertical blanking period</_long>
-		    <default>true</default>
+		    <default>false</default>
 		</option>
 	    </group>
 	    <group>
@@ -319,7 +319,7 @@
 		<option name="focus_prevention_match" type="match">
 		    <_short>Focus Prevention Windows</_short>
 		    <_long>Focus prevention windows</_long>
-		    <default>any</default>
+		    <default>!(class=Polkit-mate-authentication-agent-1)</default>
 		</option>
 	    </group>
 	    <option name="unredirect_fullscreen_windows" type="bool">
--- a/metadata/scale.xml.in
+++ b/metadata/scale.xml.in
@@ -17,7 +17,7 @@
 		<option name="key_bindings_toggle" type="bool">
 		    <_short>Key Bindings Toggle Scale Mode</_short>
 		    <_long>Key bindings toggle scale mode instead of enabling it when pressed and disabling it when released.</_long>
-		    <default>false</default>
+		    <default>true</default>
 		</option>
 		<option name="button_bindings_toggle" type="bool">
 		    <_short>Button Bindings Toggle Scale Mode</_short>
@@ -27,15 +27,12 @@
 		<option name="initiate_edge" type="edge">
 		    <_short>Initiate Window Picker</_short>
 		    <_long>Layout and start transforming windows</_long>
-		    <default>
-			<edge name="TopRight"/>
-		    </default>
 		    <allowed edgednd="true"/>
 		</option>
 		<option name="initiate_key" type="key">
 		    <_short>Initiate Window Picker</_short>
 		    <_long>Layout and start transforming windows</_long>
-		    <default>&lt;Shift&gt;&lt;Alt&gt;Up</default>
+		    <default>&lt;Super&gt;w</default>
 		</option>
 		<option name="initiate_button" type="button">
 		    <_short>Initiate Window Picker</_short>
@@ -53,6 +50,7 @@
 		<option name="initiate_all_key" type="key">
 		    <_short>Initiate Window Picker For All Windows</_short>
 		    <_long>Layout and start transforming all windows</_long>
+		    <default>&lt;Super&gt;a</default>
 		</option>
 		<option name="initiate_group_edge" type="edge">
 		    <_short>Initiate Window Picker For Window Group</_short>
@@ -96,14 +94,14 @@
 		<option name="spacing" type="int">
 		    <_short>Spacing</_short>
 		    <_long>Space between windows</_long>
-		    <default>10</default>
+		    <default>12</default>
 		    <min>0</min>
 		    <max>250</max>
 		</option>
 		<option name="speed" type="float">
 		    <_short>Speed</_short>
 		    <_long>Scale speed</_long>
-		    <default>1.5</default>
+		    <default>2.4</default>
 		    <min>0.1</min>
 		    <max>50</max>
 		    <precision>0.1</precision>
@@ -111,7 +109,7 @@
 		<option name="timestep" type="float">
 		    <_short>Timestep</_short>
 		    <_long>Scale timestep</_long>
-		    <default>1.2</default>
+		    <default>0.1</default>
 		    <min>0.1</min>
 		    <max>50</max>
 		    <precision>0.1</precision>
@@ -119,12 +117,12 @@
 		<option name="darken_back" type="bool">
 		    <_short>Darken Background</_short>
 		    <_long>Darken background when scaling windows</_long>
-		    <default>true</default>
+		    <default>false</default>
 		</option>
 		<option name="opacity" type="int">
 		    <_short>Opacity</_short>
 		    <_long>Amount of opacity in percent</_long>
-		    <default>75</default>
+		    <default>100</default>
 		    <min>0</min>
 		    <max>100</max>
 		</option>
@@ -133,7 +131,7 @@
 		    <_long>Overlay an icon on windows once they are scaled</_long>
 		    <min>0</min>
 		    <max>2</max>
-		    <default>1</default>
+		    <default>0</default>
 		    <desc>
 			<value>0</value>
 			<_name>None</_name>
--- a/metadata/decoration.xml.in
+++ b/metadata/decoration.xml.in
@@ -16,7 +16,7 @@
 	    <option name="shadow_radius" type="float">
 		<_short>Shadow Radius</_short>
 		<_long>Drop shadow radius</_long>
-		<default>8.0</default>
+		<default>15.0</default>
 		<min>0.1</min>
 		<max>18.0</max>
 		<precision>0.1</precision>
@@ -43,13 +43,14 @@
 	    <option name="shadow_y_offset" type="int">
 		<_short>Shadow Offset Y</_short>
 		<_long>Drop shadow Y offset</_long>
-		<default>1</default>
+		<default>5</default>
 		<min>-16</min>
 		<max>16</max>
 	    </option>
 	    <option name="command" type="string">
 		<_short>Command</_short>
 		<_long>Decorator command line that is executed if no decorator is already running</_long>
+		<default>/usr/bin/compiz-mate-gtk</default>
 	    </option>
 	    <option name="mipmap" type="bool">
 		<_short>Mipmap</_short>
--- a/metadata/cube.xml.in
+++ b/metadata/cube.xml.in
@@ -92,9 +92,9 @@
 		    <_short>Cube Color</_short>
 		    <_long>Color of top and bottom sides of the cube</_long>
 		    <default>
-			<red>0xfefe</red>
-			<green>0xffff</green>
-			<blue>0xc7c7</blue>
+                       <red>0xcdcd</red>
+                       <green>0xbebe</green>
+                       <blue>0x7070</blue>
 		    </default>
 		</option>
 		<subgroup>
@@ -109,9 +109,7 @@
 			<_long>List of PNG and SVG files that should be rendered on top face of cube</_long>
 			<type>string</type>
 			<hints>file;image;</hints>
-			<default>
-			    <value>freedesktop</value>
-			</default>
+			<default><value>/usr/share/images/desktop-base/debian-blueish-wallpaper-640x480.png</value></default>
 		    </option>
 		    <option name="adjust_image" type="bool">
 			<_short>Adjust Image</_short>
--- a/metadata/place.xml.in
+++ b/metadata/place.xml.in
@@ -12,7 +12,7 @@
 	    <option name="mode" type="int">
 		<_short>Placement Mode</_short>
 		<_long>Algorithm to use for window placement</_long>
-		<default>0</default>
+		<default>2</default>
 		<min>0</min>
 		<max>5</max>
 		<desc>
--- a/metadata/resize.xml.in
+++ b/metadata/resize.xml.in
@@ -40,7 +40,7 @@
 	    <option name="mode" type="int">
 		<_short>Default Resize Mode</_short>
 		<_long>Default mode used for window resizing</_long>
-		<default>0</default>
+		<default>2</default>
 		<min>0</min>
 		<max>3</max>
 		<desc>
@@ -64,20 +64,20 @@
 		<_short>Border Color</_short>
 		<_long>Border color used for outline and rectangle resize modes</_long>
 		<default>
-		    <red>0x2f2f</red>
-		    <green>0x2f2f</green>
-		    <blue>0x4f4f</blue>
-		    <alpha>0x9f9f</alpha>
+		    <red>0xffff</red>
+		    <green>0xffff</green>
+		    <blue>0xffff</blue>
+		    <alpha>0x6464</alpha>
 		</default>
 	    </option>
 	    <option name="fill_color" type="color">
 		<_short>Fill Color</_short>
 		<_long>Fill color used for rectangle resize mode</_long>
 		<default>
-		    <red>0x2f2f</red>
-		    <green>0x2f2f</green>
-		    <blue>0x4f4f</blue>
-		    <alpha>0x4f4f</alpha>
+		    <red>0x0</red>
+		    <green>0x0</green>
+		    <blue>0x0</blue>
+		    <alpha>0x1e1e</alpha>
 		</default>
 	    </option>
 	    <option name="normal_match" type="match">
--- a/metadata/fade.xml.in
+++ b/metadata/fade.xml.in
@@ -51,7 +51,7 @@
 	    <option name="window_match" type="match">
 		<_short>Fade windows</_short>
 		<_long>Windows that should be fading</_long>
-		<default>any</default>
+                <default>any &amp; !(title=notify-osd)</default>
 	    </option>
 	    <option name="visual_bell" type="bool">
 		<_short>Visual Bell</_short>
